src/vorta/application.py

```diff
@@ -113,12 +113,13 @@ def create_backup_action(self, profile_id=None):
             job = BorgCreateJob(msg['cmd'], msg, profile.repo.id)
             self.jobs_manager.add_job(job)
         else:
-            notifier = VortaNotifications.pick()
-            notifier.deliver(
-                self.tr('Vorta Backup'),
-                translate('messages', msg['message']),
-                level='error',
-            )
+            if msg['message'] != 'Current Wifi is not allowed.' or profile.show_notification_when_network_disallowed:
```

Comment:
Comparing the precise error message duplicates it and is prone to breaking, if it slightly changes later?

Reply:
Do you suggest adding a numeric value to the msg dict? Like a code or something? Vorta does not use consistent return codes, so I did not find anything.